code/utils/rf.py

```diff
@@ -0,0 +1,27 @@
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.cross_validation import KFold
+import numpy as np
+
+
+def rf_accuracy(X_train, y_train, X_test, y_test, est=1000, feat=10, depth=10):
```

Comment:
What is the motivation for limiting the tree to depth 10? I think for RF we want to grow many deep trees and its ok if any particular tree overfits, since its working with a bootstrapped version of the data